pre-docs/bolt_installing_modules.md

```diff
@@ -84,4 +84,4 @@ For more details about specifying modules in a Puppetfile, see the [Puppetfile d
      directory. To override this location, update the modulepath setting in the
      Bolt config file.
 
-
+NOTE: Bolt uses r10k to download modules, and caches them in `~/.r10k/cache` before unzipping them. If you're a Windows user in an enterprise environment, ensure `HOME` or `HOMEDRIVE + HOMEPATH` is set sensibly.
```

Comment:
Thank you so much for contributing - this will save people a lot of debugging time! One small note: I'm not sure what `sensibly` means in this context? Can you elaborate, or be more specific a bit?

Reply:
This is pretty specific to having multiple mounts and windows, so probably sensibly===on the same mount.

The issue I ran into (during a Bolt workshop) was that my HOMEDRIVE was set to a different (inaccessible) mount, than my CWD was

Reply:
Hmm, maybe "ensure `HOME` or `HOMEDRIVE + HOMEPATH` is accessible from you Bolt controller"?  It's also probably fine as is - I'll let our docs guru @cwcowellshah weigh in

Reply:
I read "sensibly" here to mean "they actually point at your HOME", which makes sense to me.